docs/reference/field-types/relationship.md

```diff
@@ -57,6 +57,14 @@ _toppings: {
 |`readOnly` | Boolean | `false` | If `true`, prevents the user from editing the field value |
 |`withRelationships` | Array |  n/a | An array of field names representing `relationship` fields you wish to populate with the connected docs. [See below](#populating-nested-relationships-using-withrelationship) for more. |
 |`withType` | String | Uses the field name, minus its leading `_` and possible trailing `s` | The name of the related type. |
+|`browse` | Boolean | `true` | If `false`, hide the browse button. |
+|`suggestionLabel` | String | `apostrophe:relationshipSuggestionLabel` | The label of the autocomplate suggestion as first entry |
```

Comment:
'autocomplate' => 'autocomplete'

How about:
'The label at the top of the autocomplete suggestions'